CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+
+## [2.0.6] - 2024-10-14
+### Breaking Changes
```

Comment:
This isn't a breaking change. As the user wouldn't need to change anything at their end. However we will recommend everyone to upgrade their version.